CONTRIBUTING.md

```diff
@@ -310,7 +310,9 @@ A good commit message should describe what changed and why.
 
 1. The first line should:
    - contain a short description of the change
-   - be 50 characters or less
+   - be 50 characters or less (or as close as possible to 50 characters if it
+     is necessary to go over in order to provide a *useful* description of the
+     change)
```

Comment:
Nit: how about remove this entire bullet point and change the first bullet point to this:

> - contain a short description of the change (preferably 50 characters or less)